authgear/authgear-sdk-js
# Implement app-initiated-sso-to-web

ref DEV-1407

## How to test

1. You should have two clients, for example `client1id` `client2id`. Set `x_app_initiated_sso_to_web_enabled` to true for both clients.
2. In the example app, fill `client1id` as the "Client ID". Fill `client2id` as the "App Initiated SSO To Web Client ID". Check "Is App Initiated SSO To Web Enabled". Press configure.
3. Press Authenticate and complete the login.
4. Now, press "App Initiated SSO To Web". You should see a browser opened and closed immediately. Then a continue screen appear.
5. Press continue, successfully logged in.
